react/src/lib/ListItem/index.js

```diff
@@ -70,19 +70,84 @@ class ListItem extends React.Component {
     parentOnSelect && parentOnSelect(e, { value, label, eventKey });
   }
 
-  handleKeyDown = (e, eventKey) => {
+  changeTabIndex = (tabbableChildren, index) => {
+    for (let i = 0; i < tabbableChildren.length; i++) {
+      if (tabbableChildren[i].tabIndex === (index === 0 ? -1 : 0)) {
+        tabbableChildren[i].tabIndex = index;
+      }
+    }
+  }
+
+  handleKeyDown = (e, eventKey, focusLockTabbableChildren, tabbableChildrenQuery) => {
```

Comment:
Getting to the point where it would make sense to just have  an opts object passed in.

Reply:
I tried, to, but it was failing builds when I use .shape